glean-core/ios/Glean/Net/HttpPingUploader.swift

```diff
@@ -122,4 +127,91 @@ public class HttpPingUploader {
         dateFormatter.setLocalizedDateFormatFromTemplate("EEE, dd MMM yyyy HH:mm:ss z")
         return dateFormatter.string(from: date)
     }
+
+    /// This function deserializes and processes all of the serialized ping files.
+    ///
+    /// This function will ignore files that don't match the UUID regex and just delete them to
+    /// prevent files from polluting the ping storage directory.
+    func process() {
+        let pingDirectory = HttpPingUploader.getOrCreatePingDirectory()
+
+        do {
+            let storageDirectory = try FileManager.default.contentsOfDirectory(
+                atPath: pingDirectory.absoluteString
+            )
+
+            for file in storageDirectory {
+                if file.lastPathComponent.matches(Constants.filePattern) {
+                    NSLog("\(Constants.logTag) : Processing ping: \(file)")
```

Comment:
Can't we rename the `logPing` method from above and reuse it so we don't have to add the `\(Constants.logTag\` in all the strings?

Reply:
The logPing method was meant to work with the GleanDebugActivity.  I would love to have a unified logging class that would hold the tag in the instance and format the output uniformly 😄, should I file a bug?

Reply:
Rather than file a bug, I think we can address this as part of work for [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1580856).